unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Utils/UserPositionMarker.cs

```diff
@@ -0,0 +1,19 @@
+﻿using UnityEngine;
+using System;
+
+internal class UserPositionMarker : IDisposable
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!